### PR TITLE
optimize images

### PR DESCRIPTION
**Pull request by paulirish**
When I was loading a draft page (https://aoe2cm.net/draft/pbxTs) i noticed the civ icons were loading veryyyyyyy slowly. 

image

This time each civ image was taking ~15 seconds to load. 

The images are much larger in filesize than they should be given their display size.

So I ran them through https://imageoptim.com/mac Afterwards, they're a fraction of the filesize they were before. 

**Before this page loaded 7.2 MB of images. Now it's just 1.6MB**

FYI: I did use lossy compression. You can try to see the changes in the github UI.. 

image

I can't spot any loss of quality with my eyes.

### CODE REVIEW

File: public/images/aoe2cm2.svg [hunk 1]
```diff
@@ -1,300 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (HSZemi) [inline public/images/aoe2cm2.svg:1]: No need to optimize this one, that's just the source file for the png

Comment (HSZemi) [reply public/images/aoe2cm2.svg:1]: I removed the file from the commit and added the rest in 74838ce2d6dc8de17a2a47d88109a7ee9adb1205

**Comment by HSZemi** [comment]
Thank you for your valuable contribution. It loads a lot faster now :yellow_heart:

**Comment by paulirish** [comment]
Cool. Sounds good!

**Comment by HSZemi** [comment]
(Don't check yet, I still have to deploy it on aoe2cm.net :sweat_smile: )

**Pull request closed without merging**